eliot-org.el

```diff
@@ -1,10 +1,26 @@
+(with-eval-after-load 'org-agenda
+  (define-key org-agenda-mode-map "e" 'org-agenda-week-view)
```

Comment:
wonder if there's an org/emacs plugin to make a list of all custom hotkeys in file(s)